oat-sa/extension-tao-proctoring
# Feature/NCC-222/implement data polling for proctoring page

**Related to task:**  https://oat-sa.atlassian.net/browse/NCC-222
**Description:** The polling data process for proctoring page is activated by setting time (ms) for `autoRefresh` field in the `config/taoProctoring/GuiSettings.conf.php`.
The data in table will be refreshed only if data are different with data in a previous response. We check delivery execution id and state status

**How to test:** 
- Set in `config/taoProctoring/GuiSettings.conf.php` interval for polling
```
'autoRefresh' => 3000,
```
- Login with proctor rights and navigate to the proctoring page
- Open new session and login as test taker
- As TT change the QTI test state by starting/pause QTI test. The data on the proctoring page should be reloaded in given poll interval

**Unit tests cli:** npx grunt testall
